### PR TITLE
Update overview-for-teams-loop

### PR DESCRIPTION
**Pull request by xjli1972**
Update Overview of Loop components in Teams

### CODE REVIEW

File: Teams/loop-components-in-teams.md [hunk 1]
```diff
@@ -81,8 +81,6 @@ If you disable these experiences as outlined in the [Settings management](/share
 
 ## Known issues
 
-- Loop components in chat cannot be edited via Office app when using Teams on Android.
-- If tenant default file permissions are set to *Specific people* (only the people the user specifies), and the sender removes some users from the *Specific people* list in the permissions dialog when creating a component, those users may still have access to the content.
 - With tenant default file permissions set to *Specific people* (only the people the user specifies), copying link to live component and pasting in another chat requires the sender to use permissions dialog and add the recipients in the Specific people option to grant access properly.
```
Comment (marco-ms) [inline Teams/loop-components-in-teams.md:84]: What does it mean "copying link to live component"?

**Review by marco-ms — APPROVED** [review]

**Review by marco-ms — APPROVED** [review]

File: Teams/loop-components-in-teams.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ Available on Teams apps on Windows, Mac, Linux, iOS, and Android.
 
 ## Loop components and .fluid files
 
-Loop components created in Teams are backed by a .fluid file stored in the creator's OneDrive. Being a file in OneDrive means that users can create, discover, and manage Loop components (.fluid files) as easily as any Office document. 
+Loop components created in Teams are backed by a .fluid (will be changed to .loop in near future) file stored in the creator's OneDrive. Being a file in OneDrive means that users can create, discover, and manage Loop components (.fluid files) as easily as any Office document.
```
Comment (JohanFreelancer9) [inline Teams/loop-components-in-teams.md:44]:
```suggestion
Loop components created in Teams are backed by a .fluid (will be changed to .loop in the near future) file stored in the creator's OneDrive. Being a file in OneDrive means that users can create, discover, and manage Loop components (.fluid files) as easily as any Office document. 
```

File: Teams/loop-components-in-teams.md [hunk 3]
```diff
@@ -81,9 +81,7 @@ If you disable these experiences as outlined in the [Settings management](/share
 
 ## Known issues
 
-- Loop components in chat cannot be edited via Office app when using Teams on Android.
-- If tenant default file permissions are set to *Specific people* (only the people the user specifies), and the sender removes some users from the *Specific people* list in the permissions dialog when creating a component, those users may still have access to the content.
-- With tenant default file permissions set to *Specific people* (only the people the user specifies), copying link to live component and pasting in another chat requires the sender to use permissions dialog and add the recipients in the Specific people option to grant access properly.
+- With tenant default file permissions set to *Specific people* (only the people the user specifies), copying link to Loop component and pasting in another chat requires the sender to use permissions dialog and add the recipients in the Specific people option to grant access properly.
```
Comment (JohanFreelancer9) [inline Teams/loop-components-in-teams.md:84]:
```suggestion
- With tenant default file permissions set to *Specific people* (only the people the user specifies), copying the link to the Loop component and pasting it in another chat requires the sender to use the permissions dialog and add the recipients in the Specific people option to grant access properly.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xjli1972, kindly commit the suggested changes. Many thanks.

**Pull request merged**